### DISCUSSION TITLE
Quarto dashboard in VS Code using quarto extension v1.111.0

### DISCUSSION THREAD

**Discussion by cmwphia** [quarto-dev/quarto-cli#9480] [Q&A] [answered]
### Description

Using Quarto extension v1.111.0 in VS Code v 1.88.1, the dashboard value for the format setting in the yaml is not being recognised: format: dashboard. ERROR: Invalid format string
Is this because the extension is behind the CLI 1.4 version, and the dashboard functionality will be available via the Quarto extension in future updates?

**Comment by mcanouil** [comment]
Which version of Quarto CLI are you using?

I cannot reproduce on macOS, using latest VSCode and Quarto extension, with either Quarto 1.4.553 or development version.
I even get autocompletion for the format.

PS: VSCode Quarto extension repository (and Visual Editor) is <https://github.com/quarto-dev/quarto>.

**Reply by cmwphia** [reply]
Yes I should have mentioned that I'm on Windows! I haven't got the CLI installed, and other functionality seems to work all OK just with the extension installed. However, I do have a colleague in my organisation who has the CLI 1.4 installed and has the same issue, though running the qmd in R Studio.

**Reply by mcanouil** [reply, marked as answer]
1.4.1 does not have dashboard. A version has three component such as 1.4.553 for the latest stable version.

Be sure to use the latest RStudio version.

For the Quarto extension to work fully and properly, Quarto CLI is required.

**Reply by cmwphia** [reply]
Thank you for your super-quick responses, that's really useful information, will install/update our versions and go from there.
